foreman_maintain.gemspec

```diff
@@ -23,6 +23,7 @@ the Foreman/Satellite up and running."
 
   s.add_dependency 'clamp'
   s.add_dependency 'highline'
+  s.add_dependency 'nokogiri'
```

Comment:
this dependency is available in epel for rpm builds

Reply:
Thought to give this a try on Satellite and here are the things,
1. The foreman-maintain is not SCLed, so it can't consume nokogiri from tfm scl.
2. There are checks in foreman-maintain itself to see if system has EPEL configured, if yes then disable it. So it wont make sense to have opposite approach to have dependency from `epel`.

This means if we want to get this merged we need to ship two packages of nokogir, one in scl and another one maybe in Satellite/Tools repository. Any thoughts on this?